### ISSUE TITLE
Release cadence page

### ISSUE THREAD

**Issue by divya-mohan0209** [suborbital/docs#4]
Need to insert a release cadence page with details regarding
- How the release cycle is scheduled for each of the projects
- The actual cadence
- The release process

**Comment by divya-mohan0209** [comment]
CC: @flaki @cohix

**Comment by flaki** [comment]
I don't think we have a regular release cadence, but it would still be useful to have a page that discusses a release cadence, and maybe shares information on new releases.

Closely related to #23 on versioning

**Comment by LauraLangdon** [comment]
From today's docs meeting notes:

> - Do we have this documented somewhere?
> - Flaki: we don’t (yet) have a fixed release cadence, but I would put this on the dev team agenda for one of next week’s calls and discuss what *can* we share about our release cadence, even if it’s not, practically, a cadence 😅
> - Flaki: note that this is closely related to Issue #23 which we also don’t yet follow (very explicitly)
> - Laura Will bring this up with the dev team on Thursday’s dev team meeting

**Comment by LauraLangdon** [comment]
At yesterday's dev meeting, we decided to revisit this if we have a regular cadence in future.
